orchestra/models.py

```diff
@@ -265,7 +268,7 @@ class Status:
         (Status.PROCESSING, 'Processing'),
         (Status.SUBMITTED, 'Submitted'))
 
-    start_datetime = models.DateTimeField(auto_now_add=True)
+    start_datetime = models.DateTimeField(default=timezone.now)
```

Comment:
open question: should we set current values to `None` and write some sort of migration script, or just move on with some dirty data for old tasks' `start_datetime`?